examples/htool_console.c

```diff
@@ -150,7 +150,12 @@ void restore_terminal(int fd, const struct termios *old_termios) {
 // in-place escape sequence processing.
 // looks for ctrl-T/q.
```

Comment:
nit: generalize this comment to say that it's looking for <kbd>CTRL</kbd> + <kbd>T</kbd> + `escape char`.

Reply:
Done